src/Tests/ServicePlan.fs

```diff
@@ -0,0 +1,19 @@
+﻿module ServicePlan
+
+open Expecto
+open Farmer
+open Farmer.Builders
+open Farmer.Arm
+open System
+
+let getResource<'T when 'T :> IArmResource> (data:IArmResource list) = data |> List.choose(function :? 'T as x -> Some x | _ -> None)
+let getResources (v:IBuilder) = v.BuildResources Location.WestEurope
+
+let tests = testList "Service Plan Tests"[
+    test "Enable zoneRedundant in service plan" {
```

Comment:
Do we also want to add a test to cover the default? (When you don't enable it the value is x?)